src/test/java/org/sagebionetworks/bridge/exporter/scheduler/BridgeExporterSchedulerTest.java

```diff
@@ -24,15 +24,19 @@
     private static final String TEST_SQS_QUEUE_URL = "test-sqs-queue";
     private static final String TEST_TIME_ZONE = "Asia/Tokyo";
 
+    private static final DateTime MOCK_NOW = DateTime.parse("2016-02-01T16:47:55.273+0900");
+
+    private static final DateTime DAILY_END_DATE_TIME = MOCK_NOW
+            .withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0).withMillisOfSecond(0).minusMillis(1);
```

Comment:
Our test and our code run the same code. If the code has a bug, our test has the same bug. We need to actually test against our expected result, not just run the same calculation in both test and code.